spec/models/post_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'associations' do
```

Comment:
- [ ] Please make sure that you add a test that a post title must not exceed 250 characters. :pray:

Reply:
Done